### ISSUE TITLE
`dotnet build` writes unnecessary information to the console

### ISSUE THREAD

**Issue by dougbu** [dotnet/cli#2326]
Console logging includes individual elapsed times (information of interest only to those writing these tools) and explanations why every dependency that was or was not built (information that's very easy to infer from the rest of the output -- there's only one reason to build). None of this is of interest to developers using the command line tools.
## Steps to reproduce
1. Clone the aspnet/Mvc repo or something else with a fair number of interdependent projects
2. Execute `./build.sh --quiet compile` or `.\build.ps1 --quiet compile`

The above command executes per-project `dotnet build` and `dotnet pack` commands out of their dependency order. (The `--quiet` part does not affect these commands because neither `--quiet` nor `--verbosity minimal` are available.)
## Expected  behaviour

A relatively quiet build with little extraneous console output. Put another way, I'd expect console logging that doesn't bury useful information. I.e. just the following for each project or dependency that's built:

```
Compiling Microsoft.AspNetCore.Mvc.Abstractions for .NETFramework,Version=v4.5.1

Compilation succeeded.
    0 Warning(s)
    0 Error(s)
```
## Actual behaviour

Output such as the following is common:

```
Project Microsoft.AspNetCore.Mvc.Abstractions (.NETFramework,Version=v4.5.1) will be compiled because some of its inputs were newer than its oldest output.
Compiling Microsoft.AspNetCore.Mvc.Abstractions for .NETFramework,Version=v4.5.1

Compilation succeeded.
    0 Warning(s)
    0 Error(s)

Time elapsed 00:00:11.1401748
```

or

```
Project Microsoft.AspNetCore.Mvc.Abstractions (.NETFramework,Version=v4.5.1) was previously compiled. Skipping compilation.
```
## Environment data

`dotnet --info` output:

```
.NET Command Line Tools (1.0.0-beta-002202)

Product Information:
 Version:     1.0.0-beta-002202
 Commit Sha:  60b23d5115

Runtime Environment:
 OS Name:     Windows
 OS Version:  6.3.9600
 OS Platform: Windows
 RID:         win81-x64
```

**Comment by Bartmax** [comment]
I would like to keep time elapsed. that's generally useful information.

but agree to be less 'versbose'. something like this?

``` 
Compiling Microsoft.AspNetCore.Mvc.Abstractions for .NETFramework,Version=v4.5.1

Compilation succeeded. (00:00:11.1401748)
    0 Warning(s)
    0 Error(s)
```

**Comment by dougbu** [comment]
@Bartmax sure.

**Comment by jstarks** [comment]
A note, golang doesn't output anything on a successful build. It feels very Windowsy for dotnet to say anything when nothing went wrong.

**Comment by piotrpMSFT** [comment]
@blackdwarf is this mitigated? we're still pretty noisy when building successfully. Should we look at the UX here?

**Comment by livarcocc** [comment]
This is our current output:

```
Microsoft (R) Build Engine version 15.1.0.0
Copyright (C) Microsoft Corporation. All rights reserved.

  Microsoft.DotNet.Cli.Utils -> /Users/livarcocc/Documents/git/cli/src/Microsoft.DotNet.Cli.Utils/bin/Debug/netstandard1.5/Microsoft.DotNet.Cli.Utils.dll
  Microsoft.DotNet.TestFramework -> /Users/livarcocc/Documents/git/cli/src/Microsoft.DotNet.TestFramework/bin/Debug/netstandard1.5/Microsoft.DotNet.TestFramework.dll
  Microsoft.DotNet.InternalAbstractions -> /Users/livarcocc/Documents/git/cli/src/Microsoft.DotNet.InternalAbstractions/bin/Debug/netstandard1.3/Microsoft.DotNet.InternalAbstractions.dll
  Microsoft.DotNet.Tools.Tests.Utilities -> /Users/livarcocc/Documents/git/cli/test/Microsoft.DotNet.Tools.Tests.Utilities/bin/Debug/netcoreapp1.0/Microsoft.DotNet.Tools.Tests.Utilities.dll
  Microsoft.DotNet.Cli.Utils.Tests -> /Users/livarcocc/Documents/git/cli/test/Microsoft.DotNet.Cli.Utils.Tests/bin/Debug/netcoreapp1.0/Microsoft.DotNet.Cli.Utils.Tests.dll
```

Are we satisfied with it?

**Comment by dougbu** [comment]
My view: @livarcocc's clip looks much better than the `dotnet build` (i.e. old world) output.

**Comment by blackdwarf** [comment]
@livarcocc looks ok to me. Is that all? I assume the "Build complete" is still there and all that?

**Comment by piotrpMSFT** [comment]
This was fixed by #4508. When invoking msbuild we now pass /v:m[inimal] to control output. Folks who want more verbose output can pass `/v:q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic]`.

**Comment by piotrpMSFT** [comment]
This change, by the way, is a very early benefit of moving to msbuild. the maturity of the logging system in msbuild is significantly ahead of where we would be with project.json today. `/v:diag` and `/pp` have proven to be invaluable tools in debugging some of the more nuanced issues posted to this repo in recent weeks.
